apps/gauzy/src/app/pages/organizations/edit-organization/edit-organization-settings/edit-organization-other-settings/edit-organization-other-settings.component.html

```diff
@@ -309,7 +309,7 @@ <h4 class="settings-section-header">
 					<nb-accordion-item-header>
 						{{ 'ORGANIZATIONS_PAGE.EDIT.ACCOUNTING' | translate }}
 					</nb-accordion-item-header>
-					<nb-accordion-item-body>
+					<nb-accordion-item-body #accountingBody [hidden]="accountingBody?.state === 'collapsed' ? true : false">
```

Comment:
```suggestion
					<nb-accordion-item-body #accountingBody [hidden]="accountingBody?.state === 'collapsed'">
```
@coderNadir please simplify condition.